app-drawer/app-drawer.html

```diff
@@ -47,8 +47,8 @@
 
 @group App Elements
 @element app-drawer
-@demo app-drawer/demo/demo1.html Simple Left Drawer
-@demo app-drawer/demo/demo2.html Right Drawer with Icons
+@demo app-drawer/demo/index.html Simple Left Drawer
```

Comment:
`app-drawer/demo/index.html` -> `app-drawer/demo/left-drawer.html` Then we can have an index.html that links to both demos. That way, people that clicked on "demo" from the standalone version of iron-component-page can access all the demos.